scripts/run-hls-test.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+set -eu
+
+# URL to the benchmark git repository and the commit to be used
+GIT_REPOSITORY=https://github.com/EECS-NTNU/hls-test-suite.git
```

Comment:
The same issue as for the llvm-test-suite script. Please use: `https://github.com/phate/hls-test-suite.git`